packages/wpcom-proxy-request/src/index.js

```diff
@@ -516,6 +517,8 @@ const wpcomAllowedOrigins = [
 	'https://calypso.localhost:3000',
 	'http://jetpack.cloud.localhost:3000',
 	'https://jetpack.cloud.localhost:3000',
+	'http://automattic.agencies.localhost:3000',
```

Comment:
I think keeping these shorter would be nice for devex, but no big deal, up to you really:

```suggestion
	'http://agencies.localhost:3000',
```

Reply:
Totally agree! Updated it here: https://github.com/Automattic/wp-calypso/pull/87292/commits/472ecc27ba96355584e4a0e37c723040618d0a66

Reply:
Updated in the diff also.